bitnami/spring-cloud-dataflow/templates/prometheus-proxy/deployment.yaml

```diff
@@ -21,9 +21,30 @@ spec:
     metadata:
       labels: {{- include "common.labels.standard" . | nindent 8 }}
         app.kubernetes.io/component: prometheus-proxy
+      annotations:
+        {{- if .Values.metrics.podAnnotations }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.metrics.podAnnotations "context" $) | nindent 8 }}
+      {{- end }}
     spec:
       {{- include "scdf.imagePullSecrets" . | nindent 6 }}
       serviceAccountName: {{ include "scdf.serviceAccountName" . }}
+      {{- if .Values.metrics.affinity }}
+      affinity: {{- include "common.tplvalues.render" (dict "value" .Values.metrics.affinity "context" $) | nindent 8 }}
+      {{- else }}
+      affinity:
+        podAffinity: {{- include "common.affinities.pods" (dict "type" .Values.metrics.podAffinityPreset "component" "prometheus-proxy" "context" $) | nindent 10 }}
```

Comment:
Wondering for default affinities, should we set component to be `prometheus-proxy`, or `server`? 

It seems make more sense if we prefer proxy to be in the same node as server. On the other hand, server's affinity setting is all servers pods prefer to stay in same node, so we can treat the logic as _all pods of the same deployment prefer to be on the same node_ .

What do you think?